cms/static/js/certificates/views/signatory_editor.js

```diff
@@ -2,18 +2,27 @@
  * This class defines an editing view for course certificates.
  * It is expected to be backed by a Certificate model.
  */
-define(['js/views/list_item_editor', 'js/certificates/models/signatory' ,"js/utils/templates", 'underscore', 'jquery', 'gettext'],
-function(ListItemEditorView, Signatory, TemplateUtils, _, $, gettext) {
+define(['js/views/list_item_editor',"js/utils/templates", 'underscore', 'jquery', 'gettext'],
```

Comment:
Ah, yes, that's right -- I forgot this file had been added in the previous PR --- I can see it now in the templates folder -- so it looks like we are good here, then.